features/feature_case/peer_access_using_driver_api/peer_access_using_driver_api.cu

```diff
@@ -0,0 +1,32 @@
+#include <cuda.h>
+#include <iostream>
```

Comment:
add a license header for the test files.

Reply:
Done